admin/acceptance_tests/learn_test.py

```diff
@@ -20,16 +22,15 @@ def insert_users_test_data(dbsession):
 def view_committing_user(request):
     from c2cgeoportal_commons.models.static import User
     user = User("momo")
-    t = request.dbsession.begin_nested()
-    request.dbsession.add(user)
-    t.commit()
+    session = DBSession()  # pylint: disable=not-callable
+    session.add(user)
     return {}
 
 
 @view_config(route_name='users_nb', renderer='./learn_test.jinja2')
 def view_displaying_users_nb(request):
     from c2cgeoportal_commons.models.static import User
-    users = request.dbsession.query(User).all()
+    users = DBSession.query(User).all()
```

Comment:
https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/sessions.html
"Once a session factory has been configured for your application, you can access session objects provided by the session factory via the session attribute of any request object." -> is there a trouble with doing so ?